catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/suggestion/LatLon.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.query.suggestion;
+
+class LatLon {
+  private final Double lat;
+  private final Double lon;
+
+  static LatLon from(Double lat, Double lon) {
```

Comment:
From what?  This naming isn't very informative

Reply:
What is the benefit to using a static method vs. a constructor?

Reply:
[WorkspaceMetacardImpl](https://github.com/codice/ddf/blob/5600fbd88f54292fb3ae84294c23cc47111bcc3a/catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/workspace/WorkspaceMetacardImpl.java#L50) has a `from` method. I think they're used when you can convert one thing to another thing when a direct mapping exists. 

In this case though a parameterized constructor would be best though

Reply:
✏️ for the static factory method